src/apps/loan-list/utils/pagination.tsx

```diff
@@ -76,11 +67,7 @@ const Pagination: FC<PaginationProps> = ({
         openModalDueDate={openModalDueDate}
         selectModalMaterial={selectModalMaterial}
       />
-      <ResultPager
-        itemsShown={displayedLoans.length}
-        hitcount={hitcount}
-        setPageHandler={setPageHandler}
-      />
+      {PagerComponent}
```

Comment:
Isn't it possible to write?:
```
<Pagercomponent />

Reply:
<img width="871" alt="image" src="https://user-images.githubusercontent.com/15377965/189346038-d29fa112-ca43-415f-908b-99020e0374a3.png">

Reply:
weird